partner_contact_birthdate/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
```

Comment:
There should be no more copyright lines in __init__.py files.

Reply:
DONE